sdk/tests/deqp/framework/common/tcuImageCompare.js

```diff
@@ -157,7 +157,7 @@ tcuImageCompare.intThresholdCompare = function(imageSetName, imageSetDesc, refer
     var compareOk = deMath.boolAll(deMath.lessThanEqual(maxDiff, threshold));
 
     if (!compareOk) {
-        debug('Image comparison failed: max difference = ' + maxDiff + ', threshold = ' + threshold);
+        testFailed('Image comparison failed: max difference = ' + maxDiff + ', threshold = ' + threshold);
```

Comment:
Per @jasomobica 's comments, let's revert these back to debug() statements and add at the bottom of es3fFragmentOutputTests.FragmentOutputCase.prototype.iterate:

  if (allLevelsOk) {
      testPassed('');
  } else {
      testFailed('One or more image comparisons failed');
  }